doc/source/getting_started/index.rst

```diff
@@ -111,11 +111,14 @@ To use PyDPF-Core, in the same Python environment, run this command:
 
 
 .. toctree::
+   :maxdepth: 2
    :hidden:
+   :caption: Introduction
```

Comment:
@luisaFelixSalles what does this render like? I think that no section title would be better for now. Until we have at least two sections.

Reply:
![image](https://github.com/user-attachments/assets/ea84832f-3023-42d5-a93e-a5f20de7cad3)